client/Details.jsx

```diff
@@ -37,7 +38,7 @@ var Details = ({ house, toggle, handleClick }) => {
           <h4 className={style.noMargin}>Appliances</h4>
           <div>
             <span className={style.greyedOut}>Appliances included: </span>
-            <div>{house.appliances.length ? house.appliances.join(', ') : 'None'}</div>
+            <div>{house.appliances && house.appliances.length ? house.appliances : 'None'}</div>
```

Comment:
@zibmcnib you should leave a PR comment on changes like this that are outside the scope of your ordinary responsibilities on SDC. I assume this is a bugfix, so what bug does this fix?